e2e/modules/environment.js

```diff
@@ -31,7 +31,7 @@ const downloadsLocation = path.join(userDataDir, 'Downloads');
 const boundsInfoPath = path.join(userDataDir, 'bounds-info.json');
 const appUpdatePath = path.join(userDataDir, 'app-update.yml');
 const exampleURL = 'http://example.com/';
-const mattermostURL = process.env.MM_TEST_SERVER_URL || 'http://localhost:8065/';
+const mattermostURL = 'https://mattermost-pr-28334.test.mattermost.cloud/'; // process.env.MM_TEST_SERVER_URL || 'http://localhost:8065/';
```

Comment:
I assume this wasn't intended to be merged?

Reply:
yeah @devinbinnie . 

sorry, I should have added a note. I'll remove it once the Dev PR gets into master